hudi-flink-datasource/hudi-flink/pom.xml

```diff
@@ -230,7 +230,8 @@
         <dependency>
             <groupId>org.apache.avro</groupId>
             <artifactId>avro</artifactId>
-            <version>${avro.version}</version>
+            <!-- Override the version to be same with Flink avro -->
+            <version>1.10.0</version>
```

Comment:
not required in this patch. but can we introduce a variable (flink.avro.version) and use that.